apps/dashboard/src/components/contract-components/contract-publish-form/default-factory.tsx

```diff
@@ -66,9 +77,36 @@ export const DefaultFactory: React.FC<DefaultFactoryProps> = ({
             }
           />
         </FormControl>
+        {shouldShowExtensionsParamInput && (
+          <>
+            <Flex flexDir="column" gap={2}>
+              <Heading size="title.md">Extension Param</Heading>
+              <Text>
+                The contract uses extensions. Choose the extension param name
+                from the initializer params below.
```

Comment:
please work with product / design to improve this copy - right now this feels like it already has the prerequisite of the user deeply understanding how it works when they read this

Reply:
++ @Yash094

Reply:
We can do a product loop for this in subsequent changes. 

For now we just want to make modular contracts deployable on testnets and get user feedback, and we're the only publishers for this.